subsys/bluetooth/controller/Kconfig

```diff
@@ -44,10 +103,34 @@ config BT_LLL_VENDOR_NORDIC
 	select BT_HAS_HCI_VS
 	select ENTROPY_NRF5_RNG
 	select ENTROPY_NRF5_BIAS_CORRECTION
+
+	select BT_CTLR_LE_ENC_SUPPORT if !BT_CTLR_PHY_2M_NRF
+	select BT_CTLR_CONN_PARAM_REQ_SUPPORT
+	select BT_CTLR_EXT_REJ_IND_SUPPORT
+	select BT_CTLR_SLAVE_FEAT_REQ_SUPPORT
+	select BT_DATA_LEN_UPDATE_SUPPORT
```

Comment:
```
select BT_CTRL_DATA_LEN_UPDATE_SUPPORT if !SOC_SERIES_NRF51X || BT_CTLR_DATA_LENGTH_CLEAR
select BT_CTLR_PRIVACY_SUPPORT if !SOC_SERIES_NRF51X
select BT_CTLR_EXT_SCAN_FP_SUPPORT
select BT_CTLR_PHY_UPDATE_SUPPORT if !SOC_SERIES_NRF51X || BT_CTLR_PHY_2M_NRF
```

Reply:
Done

Reply:
> ```
> select BT_CTRL_DATA_LEN_UPDATE_SUPPORT if !SOC_SERIES_NRF51X || BT_CTLR_DATA_LENGTH_CLEAR
> select BT_CTLR_PRIVACY_SUPPORT if !SOC_SERIES_NRF51X
> select BT_CTLR_EXT_SCAN_FP_SUPPORT
> select BT_CTLR_PHY_UPDATE_SUPPORT if !SOC_SERIES_NRF51X || BT_CTLR_PHY_2M_NRF
> ```

I think that the BT_CTLR_PHY_UPDATE_SUPPORT is always selected, since if SOC_SERIES_NRF51X  = 0, since BT_CTLR_PHY_2M_NRF depends on SOC_SERIES_NRF51X, the logical value of the if is 1 (!0 || 0 == 1); the same goes for the opposite (if SOC_SERIES_NRF51X  == 1, then !1 || 1 == 1).

So, @joerchan, I think this should remain as is. What do you think?